src/rules/no-misleading-boolean-prefixes.ts

```
@@ -0,0 +1,332 @@
+         t.type === AST_NODE_TYPES.TSBooleanKeyword ||
+         t.type === AST_NODE_TYPES.TSUndefinedKeyword ||
+         t.type === AST_NODE_TYPES.TSNullKeyword ||
+         t.type === AST_NODE_TYPES.TSVoidKeyword,
+       );
+     }
+   }
+
+   return false;
+ }
+
+ function isExpressionBooleanLike(expr: TSESTree.Expression): boolean | 'non' | 'unknown' {
```

ESLint

[error] Replace `·function·isExpressionBooleanLike(expr:·TSESTree.Expression` with `function·isExpressionBooleanLike(⏎··expr:·TSESTree.Expression,⏎`

(prettier/prettier)